### ISSUE TITLE
Performance - all routes appear as / with 10.0.3. Works fine in 9.5.5 

### ISSUE THREAD

**Issue by thekendalmiller** [airbrake/airbrake#1091]
I think this is the same issue as https://github.com/airbrake/airbrake/issues/1072

On the airbrake performance dashboard, all of the performance requests are appearing as /*path instead of the actual path. This is the page I'm talking about...removed my project_id airbrake.io/projects/#{project_id}/performance?

If I downgrade to 9.5.5, it works like I would expect and stores the request path as admin/users/show

**Comment by thekendalmiller** [comment]
Also reading the previous issue, the app I'm on also has an engine. 
I removed the routes.rb mount for the engine but it still isn't working. 
I then removed all of the routes the engine adds so that the the engine routes look like this
```
Cms::Engine.routes.draw do
 # stuff
 # commented
 # out
end
```

And airbrake works.. clearly not acceptable because I need the routes from the engine.
Just debugging and thought I'd share what I had notice in case you could point me in a direction.

This app is rails 4.2.x

**Comment by thekendalmiller** [comment]
To narrow it down further, it's one specific route in the engine that seems to break this - a catch all route. Airbrake Performance reporting works correctly as long as the engine doesn't have a catch all in it. 

```
Cms::Engine.routes.draw do
 # a
 # bunch
 # of 
 # routes
 get '*path' => 'static_contents#show', param: :path
end
```

**Comment by destari** [comment]
Thanks! We'll take a look!

**Comment by thekendalmiller** [comment]
Thanks. I was curious and spent a few more minutes.
This is the commit that introduced the issue: 6836d58d8579aaa4f367ae7c332b37e0c0b4a643
From: https://github.com/airbrake/airbrake/pull/1059

My guess is this line https://github.com/airbrake/airbrake/blob/master/lib/airbrake/rails/app.rb#L28 is the issue. It probably needs to make sure the engine doesn't recognize the route using a catchall.

**Comment by thekendalmiller** [comment]
Also, I peaked at how other APM gems get the controller and path form rails.
They are using: https://guides.rubyonrails.org/active_support_instrumentation.html#process-action-action-controller

Maybe you could get the controller and path in https://github.com/airbrake/airbrake/blob/master/lib/airbrake/rails/event.rb

**Comment by kyrylo** [comment]
@thekendalmiller thanks for the detailed description. That was really useful. I released v10.0.4 with the fix.

**Comment by thekendalmiller** [comment]
Thanks for the quick fix @kyrylo - I should have time to try it later today.
